py12box_invert/inversion_modules.py

```diff
@@ -244,12 +247,37 @@ def empirical_bayes_annualmf(self):
         
         self.analytical_gaussian_annualmf()
         
+#     def add_emissions_uncertainty(self):
+#         """
+#         Follow the approach of Rigby et al 2014 where uncertainty is added post hoc 
+#         to the emissions and mole fraction. This uncertainty is due to model error 
+#         and lifetime uncertainty. A Monte Carlo approach is taken for mf.
+#         """
+    def lifetime_var(self):
```

Comment:
We need to change this so that it's an input, rather than being hardwired. We should put the default lifetimes in a json, and I would rather this were in the project repo (e.g. py12box_agage). In that case, I guess we just need the lifetime uncertainty to be an optional argument to Invert?